application/app-chart/templates/secrets.yaml

```
@@ -0,0 +1,19 @@
+kind: Secret
+metadata:
+  name: {{ $name }}-secret
+  namespace: {{ .Release.Namespace }}
+  labels:
+    {{- include "app-chart.metadataLabels" . | nindent 4 }}
+type: Opaque
+data:
+  {{- range $index, $value := .Values.spec.env }}
+  {{- if and (regexMatch "[.]" $index) (not $allowDots) }}
+  {{- $k := $index | upper | replace "." "_" }}
+    {{ $k }}: {{ $value | default "" | toString | b64enc | default ("" | quote) }}
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 0 but found 4

(indentation)